src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -203,7 +201,30 @@ const CustomDescription = ({ amount = 0, classes, data, recipient }: CustomDescP
               </LinkWithRef>
             </>
           )}
-        </Paragraph>
+        </>
+      ) : (
+        data
+      )}
+    </Paragraph>
+  )
+}
+
+const CustomDescription = ({ amount = 0, classes, data, recipient }: CustomDescProps) => {
+  const [showTxData, setShowTxData] = useState(false)
```

Comment:
Can we move this state to `TxData` component?

Reply:
done